public/app/features/alerting/unified/components/GrafanaAlertmanagerDeliveryWarning.test.tsx

```diff
@@ -1,39 +1,20 @@
 import { render, screen, waitFor } from '@testing-library/react';
-import { setupServer } from 'msw/node';
 import React from 'react';
 import { Provider } from 'react-redux';
 
-import { setBackendSrv } from '@grafana/runtime';
-import { backendSrv } from 'app/core/services/backend_srv';
+import { setupMswServer } from 'app/features/alerting/unified/mockApi';
+import { setAlertmanagerChoices } from 'app/features/alerting/unified/mocks/server/configure';
 import { configureStore } from 'app/store/configureStore';
 
 import { AlertmanagerChoice } from '../../../../plugins/datasource/alertmanager/types';
-import { mockAlertmanagerChoiceResponse } from '../mocks/alertmanagerApi';
 import { GRAFANA_RULES_SOURCE_NAME } from '../utils/datasource';
 
 import { GrafanaAlertmanagerDeliveryWarning } from './GrafanaAlertmanagerDeliveryWarning';
+setupMswServer();
```

Comment:
The idea being that eventually, if the server was started up in `setupTests` one day then you could just remove all of the `setupMswServer` calls

Reply:
What about having our setupTests.ts :
```beforeAll(() => server.listen());
afterEach(() => server.resetHandlers());
afterAll(() => server.close());
```

and then in our jest.config.js have this (would run setupTests before each test)
```
"jest": {
  "setupFilesAfterEnv": ["<our path to this setupTests file>/setupTests.ts"]
}
```

Reply:
Jest does have [multi project support](https://jestjs.io/docs/configuration#projects-arraystring--projectconfig) so I'll look at this as well...